.changeset/ninety-onions-bow.md

```diff
@@ -0,0 +1,5 @@
+---
+'create-astro': minor
+---
+
+Automatically setup `astro check` command when configuring TypeScript
```

Comment:
```suggestion
Automatically installs the required dependencies to run the `astro check` command when `strict` or `strictest` 
 TypeScript mode is selected.
```
Since running `astro check` requires both `@astrojs/check` and `typescript`, maybe makes sense to say "required dependencies" (plural) here, because these will be added to your project?

Reply:
I do agree. But I have to adjust some wording at the end because `astro check` will be applied to all TypeScript options (`relaxed`, `strict`, `strictest`).

Let me know if this change looks good to you.